Doc/library/sqlite3.rst

```diff
@@ -2467,10 +2467,9 @@ It is suggested to set *autocommit* to ``False``,
 which implies :pep:`249`-compliant transaction control.
 This means:
 
-* :mod:`!sqlite3` ensures that a transaction is always open,
-  so :meth:`Connection.commit` and :meth:`Connection.rollback`
-  will implicitly open a new transaction immediately after closing
-  the pending one.
+* :func:`connect`, :meth:`Connection.commit`, and :meth:`Connection.rollback`
+  implicitly open a new transaction to ensure that a transaction is always
+  open.
```

Comment:
See my comment on the issue: https://github.com/python/cpython/issues/99824#issuecomment-1328339774

If we are to mention this explicitly, we should do so in the `sqlite3.connect` reference. Also, it is nice if we can discuss the problem and how to solve it on the issue first. When the discussion has landed, a PR may be created unless the conclusion was to leave things as they are.